deployment/vars/openzaak.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# Overrides of role defaults
+
+openzaak_secret_key: 'og+88npjwk9ve!$5+1mdeyndp%94)09(re%f39rfqm$53n8)bk'
+
+openzaak_db_name: openzaak-test
+openzaak_db_host: "{{ db.host }}"
+openzaak_db_port: "{{ db.port }}"
+openzaak_db_username: openzaak-test
+openzaak_db_password: pdm01688rzvzxaemv3c6x5thl3bhrox6
```

Comment:
Ahm, moet dit niet encrypted zijn?

Reply:
Yep, I fucked up. I'm rotating the secrets and encrypting it, and I'll write a git hook to prevent this. Been bitten too often by this already now.